question/tests/behat/columnsortorder.feature

```diff
@@ -0,0 +1,47 @@
+@core @core_question @javascript
+Feature: An plugin column can be reordered and displayed in the question bank view.
+  In order to reorganise the question bank view columns
+  As an admin
+  I need to be able to reorder them
+
+  Background:
+    Given the following "users" exist:
+      | username | firstname | lastname | email |
+      | teacher1 | Teacher | 1 | teacher1@example.com |
+    And the following "courses" exist:
+      | fullname | shortname | format |
+      | Course 1 | C1 | weeks |
+    And the following "course enrolments" exist:
+      | user     | course | role           |
+      | teacher1  | C1     | editingteacher |
+    And the following "question categories" exist:
+      | contextlevel | reference | name           |
+      | Course       | C1        | Test questions |
```

Comment:
The question category should created in the quiz context, so you will need to move this after the quiz creation and add the right context.

Reply:
Question category created in `Activity module`